tests/test_utils/test_common_utils.py

```diff
@@ -119,23 +119,20 @@ def test_relation_utils(self):
         assert expected_rrel_target.value == rrel_trg.value
         assert expected_empty.is_valid() is False
 
-        expected_nrel_target = get_element_by_norole_relation(src, nrel_node)
-        expected_empty = get_element_by_norole_relation(src, rrel_node)
+        expected_nrel_target = search_element_by_non_role_relation(src, nrel_node)
+        expected_empty = search_element_by_non_role_relation(src, rrel_node)
         assert expected_nrel_target.is_valid()
         assert expected_nrel_target.value == nrel_trg.value
         assert expected_empty.is_valid() is False
 
-    def test_get_system_idtf(self):
+    def test_get_element_system_identifier(self):
         test_idtf = "rrel_1"
         test_node = ScKeynodes[test_idtf]
-        assert get_system_idtf(test_node) == test_idtf
+        assert get_element_system_identifier(test_node) == test_idtf
 
     def test_deletion_utils(self):
-        src, rrel_trg, nrel_trg = create_nodes(sc_types.NODE_CONST, sc_types.NODE_CONST, sc_types.NODE_CONST)
-        rrel_edge = create_binary_relation(sc_types.EDGE_ACCESS_CONST_POS_PERM, src, rrel_trg)
-        nrel_edge = create_norole_relation(src, nrel_trg)
-        assert delete_edges(src, rrel_trg, sc_types.EDGE_ACCESS_VAR_POS_PERM)
-        assert delete_elements(nrel_edge, src, rrel_trg, nrel_trg)
-
-        result = client.check_elements(rrel_edge)[0]
-        assert result.is_valid() is False
```

Comment:
why did you remove these 2 lines?

Reply:
it throw exception, because element is not valid

Reply:
but previously unknown type was returned for not valid elements